index.html

```diff
@@ -1,227 +1,293 @@
-            </div>
-            <div class="donut"></div>
-            <div class="third">
-                <div class="square3"></div>
-                <div class="donut2"></div>
-            </div>
-            <div class="fourth">
-                <div class="semicircle2"></div>
-                <div class="donut3"></div>
-            </div>
-            <div class="fifht">
-                <div class="circle2"></div>
-                <div class="square4"></div>
-            </div>
-            <div class="donut4"></div>
-            <div class="donut5"></div>
-        </div>
-    
-        <div class="welcome">
-            <div class="top">
-                <div class="header">
-                    <p class="logo">My Logo</p>
-                </div>
-                <div class="menuhover">
-                    <ul>
-                        <li><a href="#portfolio">Portfolio</a></li>
-                        <li><a href="#about">About</a></li>
-                        <li><a href="#Contact">Contact</a></li>
-                    </ul>
-                    <button>
-                         <i class="fa-solid fa-bars"></i>
-                    </button>
-                </div>
-            </div>
-            <div class="intro">
-                <p class="text">Im Gerson Higgins. Glad to see you!</p>
-                <p class="text2">I'm a software developer! I can help you build a product ,
-                     feature or website Look through some of my work and experience! If you like 
-                     what you see and have a project you need coded, don't hesitate to contact me.</p>
-                <p class="connect">LET'S CONNECT</p>
-                <div class="icons">
-                    <div class="icon1"><i class="fa-brands fa-twitter"></i></div>
-                    <div class="icon2"><i class="fa-brands fa-linkedin-in"></i></div>
-                    <div class="icon3"><i class="fa-regular fa-m"></i></div>
-                    <div class="icon4"><i class="fa-brands fa-github"></i></div>
-                    <div class="icon5"><i class="fa-regular fa-hand-peace"></i></div>
-                </div>
-            </div>
-        </div>    
+      <div class="background">
+        <div class="first">
+          <div class="square"></div>
+          <div class="semicircle"></div>
+        </div>
+        <div class="second">
+          <div class="square2"></div>
+          <div class="circle"></div>
+        </div>
+        <div class="donut"></div>
+        <div class="third">
+          <div class="square3"></div>
+          <div class="donut2"></div>
+        </div>
+        <div class="fourth">
+          <div class="semicircle2"></div>
+          <div class="donut3"></div>
+        </div>
+        <div class="fifht">
+          <div class="circle2"></div>
+          <div class="square4"></div>
+        </div>
+        <div class="donut4"></div>
+        <div class="donut5"></div>
+      </div>
```

Comment:
- [ ] There's horizontal scrollbar on both the desktop and mobile versions of your design, kindly rmove it.

[scrolbar.webm](https://user-images.githubusercontent.com/26597432/228668655-59399c9b-8300-4463-ad30-f9293031df09.webm)


- [OPTIONAL] Also, I noticed you made the background image yourself. You don't need to do that, you only need to download the background image and add it to your section via background-image property from your css file.

[your-make.webm](https://user-images.githubusercontent.com/26597432/228668915-5392bab7-f5df-4a17-a631-33227ae5abc4.webm)